test/check-bin.sh

```diff
@@ -19,7 +19,10 @@ expectOut () {
   expect=$2
   msg=$3
 
-  if [ $expect != $actual ]; then
+  nodeV=`node -v`
+  
+  # if Node 0.10.*, node will sometimes exit with status 8 when an error is thrown
+  if [[ $expect != $actual || $nodeV == v0.10.* && $expect == 1 && $actual == 8 ]] ; then
```

Comment:
is this how precedence works here? `[[ $expect != $actual || [[$nodeV == v0.10.* && $expect == 1 && $actual == 8]] ]]` can we make that explicit?